app/templates/services/service_submission.html

```diff
@@ -7,32 +7,40 @@
 {% import "macros/submission.html" as submission %}
 
 {% block breadcrumb %}
-  {%
-    with items = [
+  {% with items = [
       {
-        "link": "/",
-        "label": "Digital Marketplace"
+        "text": "Digital Marketplace",
+        "href": "/"
       },
       {
-        "link": url_for(".dashboard"),
-        "label": "Your account"
+        "text": "Your account",
+        "href": url_for('.dashboard')
       },
       {
-        "link": url_for(".framework_dashboard", framework_slug=framework.slug),
-        "label": "Apply to " + framework.name
+        "text": "Apply to " + framework.name,
+        "href": url_for(".framework_dashboard", framework_slug=framework.slug)
       },
       {
-        "link": url_for(".framework_submission_lots", framework_slug=framework.slug),
-        "label": "Services"
-      },
-      {
-        "link": url_for(".framework_submission_services", framework_slug=framework.slug, lot_slug=service_data.lot),
-        "label": service_data.lotName
-      } if not lot.oneServiceLimit else {}
+        "text": "Services",
+        "href": url_for(".framework_submission_lots", framework_slug=framework.slug)
+      }
     ]
   %}
-    {% include "toolkit/breadcrumb.html" %}
+    {% if lot.oneServiceLimit %}
+      {{ govukBreadcrumbs({"items": items + [{"text": service_data.get('serviceName', service_data['lotName'])}]}) }}
+    {% else %}
+      {{ govukBreadcrumbs({"items": items + [
+        {
+          "text": service_data.lotName,
+          "href": url_for(".framework_submission_services", framework_slug=framework.slug, lot_slug=service_data.lot)
+          },
+        {
+          "text": service_data.get('serviceName', service_data['lotName'])
+        }
+      ]}) }}
```

Comment:
Could we maybe refactor this to just add the extra items to the existing array or items and just have one call to the `govukBreadcrumbs` macro. Could all this computation/logic etc. be done in the view? The template could be simplified to use "items" that the view defines.

Example of where we do that https://github.com/alphagov/digitalmarketplace-briefs-frontend/blob/master/app/templates/buyers/award.html#L7-L13 the view defines what `breadcrumbs` is.

Reply:
Actually, could we follow this pattern across all the templates so that we are consistent across our frontend repos

Reply:
I think there is definitely ways that we could improve the way we implement breadcrumbs, I think it probably needs a spike though to look at different options and make a decision. Could you create a tech debt ticket for this?